x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/constants.ts

```diff
@@ -292,6 +292,48 @@ export const SOURCE_CONFIG_DESCRIPTION = i18n.translate(
   }
 );
 
+export const SYNC_MANAGEMENT_TITLE = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.sync.management.title',
+  {
+    defaultMessage: 'Sync management',
+  }
+);
+
+export const SYNC_MANAGEMENT_DESCRIPTION = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.sync.management.description',
+  {
+    defaultMessage: 'Enable and disable extraction of specific content for this source.',
+  }
+);
+
+export const SYNC_MANAGEMENT_SYNCHRONIZE_LABEL = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.sync.management.synchronize.label',
+  {
+    defaultMessage: 'Synchronize this source',
+  }
+);
+
+export const SYNC_MANAGEMENT_THUMBNAILS_LABEL = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.sync.management.thumbnails.label',
+  {
+    defaultMessage: 'Sync thumbnails',
+  }
+);
+
+export const SYNC_MANAGEMENT_THUMBNAILS_GLOBAL_CONFIG_LABEL = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.sync.management.thumbnails.globalConfig.label',
+  {
+    defaultMessage: 'Sync thumbnails - disabled at global configuration level',
+  }
+);
+
+export const SYNC_MANAGEMENT_CONTENT_EXTRACTION_LABEL = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.sync.management.content_extraction.label',
```

Comment:
```suggestion
  'xpack.enterpriseSearch.workplaceSearch.contentSources.syncManagementContentExtractionLabel',
```
We typically use the 4th segment here to namespace in the correct app section. Also prefer to camelCase the last item per the [docs](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md)